src/Hello/Hello.cs

```diff
@@ -9,16 +9,27 @@
 // The log level for the logger factory
 LogLevel logLevel = Enum.Parse<LogLevel>(Environment.GetEnvironmentVariable("LOG_LEVEL") ?? "Debug");
 
-// The path to the server certificate used for TLS.
-string serverCert = Environment.GetEnvironmentVariable("SERVER_CERT") ?? "/certs/server.p12";
+// The private key for the server certificate.
+string serverKey = Environment.GetEnvironmentVariable("SERVER_KEY") ?? "/certs/privkey.pem";
+
+// The server certificate (with the full chain) file.
+string serverCert = Environment.GetEnvironmentVariable("SERVER_CERT") ?? "/certs/fullchain.pem";
```

Comment:
and `/certs/server_cert.pem`

Reply:
`fullchain.pem` is what's used by let's encrypt. I'd prefer to keep it.

Reply:
What about changing the env variable names then?